docs/plans/2026-02-20-eval-framework-phase-3.plan.md

```
@@ -0,0 +1,139 @@
+   - Add guard: `if [[ ${#PLAN_TASKS[@]} -eq 0 ]]; then ... fi`
+
+**Dependencies:** None
+**Parallelizable:** Yes
+
+---
+
+## Parallelization
+
+All 4 streams are independent:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)